go/cmd/remotetool/main.go

```diff
@@ -29,35 +36,28 @@ type OpType string
 const (
 	downloadActionResult OpType = "download_action_result"
 	showAction           OpType = "show_action"
+	downloadBlob         OpType = "download_blob"
+	downloadMerkleTree   OpType = "download_merkle_tree"
 )
 
 var (
-	operation  = flag.String("operation", "download_action_result", "Specifies the operation to perform. Currently only download_action_result is supported.")
+	operation  = flag.String("operation", "", "Specifies the operation to perform. Currently only download_action_result is supported.")
```

Comment:
I think this should be removed and the appropriate options should be added.

Reply:
Done.